### DISCUSSION TITLE
Student Developer Pack Application

### DISCUSSION THREAD

**Discussion by Kaustubh4S** [community/community#68021] [GitHub Education] [answered]
### Hi everyone,

Recently, I applied for student Developer Pack on GitHub on September 20. It says that I applied and approved but I didn't get to access the developer pack till now. Please help me on this topic.

**Comment by arafajs** [comment]
Hi @Kaustubh4S 
According to the GitHub documentation, the activation process of Benefits for the GitHub Student Developer Pack usually takes up to 72 hours. However, it may take longer during peak times, such as during the start of a new semester.

**Reply by Kaustubh4S** [reply]
It's already been 72 hours plus it is not start of semester

**Comment by mantasu** [comment, marked as answer]
Quoting from [support page](https://support.github.com/request/education?tags=docs-generic):

> Applications may take several weeks to be approved. We’re working as fast as we can but the review process can take a while especially during times where we receive high volumes of applications, such as the start of a new semester.

If it says approved and it is green that means you still have to wait for a bit before you get access to the benefits. Once the "Approved" tag is purple, it should be ready:

![Screenshot 2023-09-24 072637](https://github.com/community/community/assets/56790921/30412a58-0e47-4b3c-8f87-fe9c85ec62bb)

Otherwise, after a few of weeks if you still don't have access, you may want to open a [ticket](https://support.github.com/request/education?tags=docs-generic).

P.S. It is the start of the semester for the majority of the universities and courses worldwide, therefore, this period is very busy for the staff.

**Reply by Kaustubh4S** [reply]
Thank you it's finally purple.

**Reply by eagui041** [reply]
where did you go to check status?

**Reply by Kaustubh4S** [reply]
https://education.github.com/globalcampus/student

Here you will find the your current status

**Reply by samus-aran** [reply]
Thanks @mantasu for answering 🎉 & thank you @Kaustubh4S for returning back and letting us all know your status.

We appreciate all the community's patience as we work through these requests ❤️

**Comment by sinister-virus** [comment]
I have not received any benefits in the last 10 months.
The approved is Purple, but why it hasn’t turned green yet?
![Screenshot 2024-04-26 154421](https://github.com/community/community/assets/51523645/b095a030-8cf1-4b4a-b59d-ecb96725a3fd)
![Screenshot 2024-04-26 154434](https://github.com/community/community/assets/51523645/592a1319-761d-488e-9750-163384e8dbb3)
